WordPressKit/RemotePlanDetail.swift

```diff
@@ -0,0 +1,41 @@
+import Foundation
+
+@objc public class RemotePlanDetail: NSObject, Codable {
```

Comment:
The name of this class is confusing. Initially I thought it would be details for the `RemotePlan` class. Boy was I wrong! 😉 I would request two things:
1. Rename this to something more specific, like `RemotePlanApiVersion1_3` (or something less crappy).
2. Add some comments as to why this exists and why we're not using `RemotePlan`.